src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -68,6 +71,9 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
         }
     });
 
+    folder && await gitPull(folder.uri);
```

Comment:
I think you could just change `gitPull` to accept `repo` as a paramemter and pass it `context.repo` instead.